src/fmu/dataio/_models/fmu_results/specification.py

```diff
@@ -171,6 +171,19 @@ class FaultRoomSurfaceSpecification(BaseModel):
     """A name id of the faultroom usage."""
 
 
+class TriangulatedSurfaceSpecification(BaseModel):
+    """Specifies relevant values describing a triangulated surface."""
+
+    name: str
+    """Name of the triangulated surface."""
```

Comment:
Not sure if the `name`  belongs in the specification, for most datatypes we we rely on the object having a `name` attribute and then that is added to the metadata as `name` there.

I see we now have the `name` stored under the `header` attribute for the TriangulatedSurface, but maybe this could be changed to make the existing logic work for this datatype as well? 🙂